contracts/modules/common/Utils.sol

```diff
@@ -75,23 +75,29 @@ library Utils {
             if(
```

Comment:
For extracting the `methodId` shall we use the internal `functionPrefix` function we already have in the library?

Reply:
We could but `functionPrefix` contains an unnecessary data size check and overall slightly increases our gas cost (by 24 to 93 gas in assets.js tests). Since `recoverSpender` is called in every subcall, I'd lean towards saving that small cost.